### PR TITLE
Correct the LICENSE text.

### PR DESCRIPTION
**Pull request by nenaddedic**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2018 Google LLC
+   Copyright [yyyy] [name of copyright owner]
```
Comment (aysylu) [inline LICENSE:190]: Is this intentional, to remove the year and company?

Comment (nenaddedic) [reply LICENSE:190]: Yes.  Note that this is in the appendix starting at L179, and is part of the cut-and-paste template that one can use to apply the license to their work.  IOW, this part of the text applies to the license itself, not to Kritis.

Comment (aysylu) [reply LICENSE:190]: Oh I see now, thank you for clarifying.

**Comment by aysylu** [comment]
The failing kokoro test is unrelated to the PR.

**Pull request merged**